### PR TITLE
Add export key

### PR DESCRIPTION
**Pull request by sbailey-arm**
Also tweaked opcodes for asymmetric encrypt and decrypt

### CODE REVIEW

File: src/parsec_client/operations/psa_export_key.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+For standard key types, the output format is as follows:
+
+- For symmetric keys, including [`MAC`](psa_key_attributes.md#hmac-type) keys, the format is the raw
+   bytes of the key.
+- For [`DES`](psa_key_attributes.md#des-type), the key data consists of 8 bytes. The parity bits
+   must be correct.
+- For [`Triple-DES`](psa_key_attributes.md#des-type), the format is the concatenation of the two or
+   three DES keys.
+- For RSA key pairs, with key type [`RsaKeyPair`](psa_key_attributes.md#rsakeypair-type), the format
```
Comment (hug-dev) [inline src/parsec_client/operations/psa_export_key.md:34]: For RSA, could you do the same than in [PsaExportPublicKey](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/psa_export_public_key.html) with a note (`[1]`) indicating the format? And also a link to the RFC on "RFC 8017" as in the spec 😬

File: src/parsec_client/operations/psa_export_key.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+   is the non-encrypted DER encoding of the representation defined by PKCS#1 in RFC 8017 as
```
Comment (ionut-arm) [inline src/parsec_client/operations/psa_export_key.md:35]: Could you add a link here for [RFC 8017](https://tools.ietf.org/html/rfc8017#appendix-A.1.2) and one below for [RFC 5915](https://tools.ietf.org/html/rfc5915), like the one for RFC 7748?

File: src/parsec_client/operations/psa_export_key.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+   RSAPrivateKey, version 0.
+- For elliptic curve key pairs, with key type [`EccKeyPair`](psa_key_attributes.md#ecckeypair-type),
+   the format is a representation of the private value.
+   - For Weierstrass curve families sectXX, secpXX, FRP and Brainpool, the content of the privateKey
```
Comment (hug-dev) [inline src/parsec_client/operations/psa_export_key.md:39]: Can you put the same words that are in fixed-width font in the spec inside back-ticks here? And also add links on the RFC as it is done in the spec?

**Comment by hug-dev** [comment]
Re-starting CI

**Comment by ionut-arm** [comment]
It wasn't added in the left hand menu -_-

**Pull request merged**